admin/pre_reg/utils.py

```diff
@@ -0,0 +1,36 @@
+
+SORT_BY = {
```

Comment:
This is a consolidation so that the template renderer doesn't need to deal with `-` since this dict is used in sorting the lists.